<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Dominic Canare &lt;dom@greenlightgo.org&gt;</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="titleInput">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="locationInput">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Date / time</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCalendarWidget" name="dateInput">
          <property name="horizontalHeaderFormat">
           <enum>QCalendarWidget::SingleLetterDayNames</enum>
          </property>
          <property name="verticalHeaderFormat">
           <enum>QCalendarWidget::NoVerticalHeader</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTimeEdit" name="startTimeInput">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="displayFormat">
             <string>hh:mm AP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string> to </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="stopTimeInput">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTextEdit" name="descriptionInput">
        <property name="autoFormatting">
         <set>QTextEdit::AutoAll</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Registration limit</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="registrationLimitInput">
        <property name="toolTip">
         <string>For no limit, set to 0</string>
        </property>
        <property name="statusTip">
         <string>The maximum number of people who are allowed to register. For no limit, set to 0.</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Prices</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <layout class="QGridLayout" name="priceGrid"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Resources</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <layout class="QGridLayout" name="resourceGrid"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Post to</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <layout class="QGridLayout" name="postToGrid"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="publishButton">
      <property name="text">
       <string>Publish</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>477</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionReload"/>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuPlugins"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReload">
   <property name="text">
    <string>Reload plugins</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
